registry.json

```diff
@@ -1,4 +1,19 @@
 [
+  {
+    "ticker": "ROSE",
+    "poolId": "fffda60991eafa48674f7137a76e11ddc2464e94c73a4319629072b4",
+    "social": {
+      "facebook": "",
+      "twitter": "RoseStaking",
+      "website": "https://github.com/RosePoolStaking/stakepool",
+      "reddit": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "t.me/rosestaking",
```

Comment:
Please use a real telegram handle. No groups or channels. 

Please do not include the "t.me"